src/TheSpace/HarbergerMarket.sol

```diff
@@ -155,7 +164,7 @@ contract HarbergerMarket is ERC721, IHarbergerMarket, Multicall, AccessRoles {
     /**
      * @dev See {IERC20-totalSupply}. Always return total possible amount of supply, instead of current token in circulation.
      */
-    function totalSupply() public view returns (uint256) {
+    function totalSupply() public view override(ERC721Enumerable) returns (uint256) {
         return _totalSupply;
     }
```

Comment:
Add these too?

```solidity
    function supportsInterface(bytes4 interfaceId)
        public
        view
        override(ERC721, ERC721Enumerable)
        returns (bool)
    {
        return super.supportsInterface(interfaceId);
    }
```

Reply:
Yes, I will update this.

Reply:
> Yes, I will update these.

Oh, I just find that I have added `override ERC721Enumerable` in  `supportsInterface`, see https://github.com/gary02/contracts/blob/5f1cd251a5d9efb94cbce9796010d4fa9206e966/src/TheSpace/HarbergerMarket.sol#L107